### PR TITLE
SG-35069 - Rebranding - Change all the shotgrid/shotgun occurrences in the repository

### PR DESCRIPTION
**Pull request by juanburgosautoglb**
Change all the shotgrid/shotgun occurrences in the repository.

### CODE REVIEW

File: README.md
Comment (carlos-villavicencio-adsk) [inline README.md:1]: Please remove the Python 2.7 badge. You can copy this line from tk-core.

File: README.md [hunk 1]
```diff
@@ -14,10 +14,11 @@ This repo has got several branches, illustrating slightly different workflows:
 
 - the `master` branch contains a standard multi app which runs in all engines.
 - the `shotgun_multi_select` branch shows how to get started if you want to create
-  a *shotgun specific* app where you can select multiple objects in ShotGrid and
-  pass this selection into the app.
-- the `shotgun_model_delegate` branch illustrates how to use the ShotGrid MVC model
-  for efficient data retrieval and the toolkit widget delegate system for display of content.
+  a *shotgun specific* app where you can select multiple objects in
```
Comment (carlos-villavicencio-adsk) [inline README.md]:
```suggestion
  a *toolkit specific* app where you can select multiple objects in
```

I'm not sure about this. @julien-lang what do you think?

**Review by carlos-villavicencio-adsk — CHANGES_REQUESTED** [review]
Please add a [SECURITY.md](https://github.com/shotgunsoftware/tk-core/blob/master/SECURITY.md) file.

File: README.md (hunk 1 above)
Comment (julien-lang) [reply README.md]: I think it should be:
```suggestion
  a *Flow Production Tracking Toolkit specific* app where you can select multiple objects in
```

**Review by carlos-villavicencio-adsk — APPROVED** [review]
LGTM

File: README.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
-[![Python 2.7 3.7](https://img.shields.io/badge/python-2.7%20%7C%203.7-blue.svg)](https://www.python.org/)
+[![Python](https://img.shields.io/badge/python-3.11%20%7C%203.10%20%7C%203.9%20%7C%203.7-blue.svg)](https://www.python.org/)
```
Comment (julien-lang) [inline README.md]: well, the problem is we don't know ... we need to test.

**Review by julien-lang — APPROVED** [review]
Let's make this PR "rebrand-only". We will create other ones for updating support(python version)

**Pull request merged**